templates/scenarios/notification-restify-ts/bot/src/index.ts

```diff
@@ -7,7 +7,7 @@ import { CardData } from "./cardModels";
 // Create HTTP server.
 const server = restify.createServer();
 server.listen(process.env.port || process.env.PORT || 3978, () => {
-  console.log(`\nBot Started, ${server.name} listening to ${server.url}`);
+  console.log(`\nApp Started, ${server.name} listening to ${server.url}`);
```

Comment:
Any change to `import { bot } from "./internal/initialize";` ?

Reply:
Thanks, updated